html/cross-origin-opener-policy/coep-navigate-popup.https.html

```diff
@@ -49,11 +49,12 @@
     const channel = title.replace(/ /g,"-");
     const navigateHost = site === "same-origin" ? SAME_ORIGIN : SAME_SITE;
     const navigateURL = `${navigateHost.origin}/html/cross-origin-opener-policy/resources/coop-coep.py?coop=${variant.coop}&coep=${variant.coep}&channel=${channel}`;
-    const opener = site === "same-origin" ? variant.opener : false;
+    const opener = site === "same-origin" ? variant.opener : true;
```

Comment:
I don't understand why navigating a COOP + COEP popup to a unsafe-none document would not break the opener. I noticed this in at least one more test below as well.

Reply:
I believe it should break the opener. So to clarify, the popup first has COOP: same-origin + COEP: require-corp, and is then navigated to `navigateURL`. When `site` is not "same-origin", this navigation nulls the opener, and it doesn't matter what happens later; the opener will still be null.

Fixing.

Reply:
Fixed in https://github.com/web-platform-tests/wpt/pull/20874/commits/1aeef8bba02f36fdca37fb5f483ccd11b6dfd462

Reply:
@annevk 

> I noticed this in at least one more test below as well.

Can you clarify where?